src/cli.rs

```diff
@@ -28,10 +28,11 @@ This will prompt you for an API Key that can be generated in Apollo Studio.
 
 The most common commands from there are:
 
-    - rover graph push: Push a schema to Apollo Studio
-    - rover graph check: Validate changes to a graph
+    - rover graph fetch: Fetch a graph schema from the Apollo graph registry
```

Comment:
I opted to talk about this as "Apollo graph registry" instead of "Apollo Studio" since I think of studio and rover to both be frontends to the Apollo graph registry. Happy to be persuaded otherwise!